rca/project_styleguide/templates/patterns/molecules/navigation/primarynav.html

```diff
@@ -81,6 +83,7 @@
                                     data-menu-id="{{ forloop.parentloop.counter}}-{{ forloop.counter }}"
                                     data-menu="{{ forloop.parentloop.counter }}-{{ forloop.counter }}"
                                     {% if item.tertiary_links %}data-menu-parent{% endif %}
+                                    role="menuitem"
```

Comment:
same query as the other PR - do we definitely want to add these roles?

Reply:
I added them as the parent `<ul>`'s had a role of `menu` and Wave was reporting broken ARIA menu's without them. I hadn't realised these had all been removed from wagtail kit - what was the reason for that? I guess the other option is to remove all the roles from the menu entirely.

Reply:
I think Thibaud said they were incorrect or unhelpful. Might be worth double checking with him if you have time, otherwise if it is ok now for the automated checks it should be fine to leave as is I think.